superset/databases/api.py

```diff
@@ -237,6 +236,50 @@ class DatabaseRestApi(BaseSupersetModelRestApi):
         ValidateSQLResponse,
     )
 
+    @expose("/<int:pk>/connection", methods=("GET",))
+    @protect()
+    @safe
+    def get_connection(self, pk: int) -> Response:
+        """Get database connection info.
+        ---
+        get:
+          description: >-
```

Comment:
nit: I'm currently switching the description for summary, looks better on swagger